app/models/tenant.rb

```diff
@@ -1,9 +1,17 @@
 class Tenant < ApplicationRecord
+  scope :current, ->(user) { where(:external_tenant => user.tenant) }
+
   validates :external_tenant, :uniqueness => true, :presence => true
 
   before_validation :setup_settings, :unless => proc { settings.nil? }
   after_initialize :setup_settings, :unless => proc { settings.nil? }
 
+  def self.scoped_tenants
+    current(ManageIQ::API::Common::Request.current.user)
```

Comment:
@syncrou Should we just raise an error if the current.user doesn't exist. Because we should never be running without a request

Reply:
I added the `rescue` so the method will mimic how `ActiveRecord` returns an empty `[]` when there are no results. 

The thought was, an error gives the caller ( external calller ) too much information about what they didn't include, where the empty array simply doesn't return anything.

Reply:
👍